README.md

```diff
@@ -3121,6 +3124,13 @@ before do
 end
 ```
 
+You can ensure a block of code runs after every request (including failures) with `finally`:
+```ruby
+finally do
+  This.block(of: code) # Will definetely run after every request
```

Comment:
maybe just `# this code will run after every successful or failed request`, cause `This.block(of: code)` is not ruby